packages/client/README.md

```diff
@@ -93,13 +93,15 @@ Fetch the number of business majors per-capita in ([California](https://datacomm
 
-| Method           | Output                                                                                       | Description                                                                                                                 |
-| ---------------- | -------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------- |
-| getCsv           | `string`                                                                                     | CSV string with most recent data observations for the specified entities and variables                                      |
-| getCsvSeries     | `string`                                                                                     | CSV string with all data observations for the specified entities and variables                                              |
-| getDataRows      | `Record<string, string \| number \| boolean \| null>[]`                                      | Data row objects with most recent data observations for the specified entities and variables                                |
-| getDataRowSeries | `Record<string, string \| number \| boolean \| null>[]`                                      | Data row objects with all data observations for the specified entities and variables                                        |
-| getGeoJson       | Promise<[FeatureCollection](https://www.jsdocs.io/package/@types/geojson#FeatureCollection)> | GeoJSON feature collection with place geometries and most recent data observations for the specified entities and variables |
+| Method                     | Output                                                                                       | Description                                                                                                                                                                 |
+| -------------------------- | -------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| getCsv                     | `string`                                                                                     | CSV string with most recent data observations for the specified entities and variables                                                                                      |
+| getCsvGroupedByEntity      | `string`                                                                                     | CSV string with the most recent data observations for the specified entities and variables. Results are grouped by entity with all variable observations on a single row.   |
```

Comment:
The current descriptions don't make it clear to me exactly how "getCsv" and "getCsvGroupedByEntity" are different. Is "getCsv" variables on the rows and entities on the columns? Might be helpful to also have a descriptor of how results in "getCsv" are laid out so it's more clear what grouped by entity means.

I'd also specify that getCsvGroupedByEntities returns one entity per row for clarity.

Reply:
tried to add some clarity here